### PR TITLE
chore(publish): publish alpha version

### PR DESCRIPTION
**Pull request by sdodson99**
**changes**

* migrated to peer dependencies
* updated package metadata
* added publish script
* added publish docs

also published an alpha version on npm: https://www.npmjs.com/package/@headless-comments/react

### CODE REVIEW

File: package/package.json [hunk 1]
```diff
@@ -29,11 +45,12 @@
...
+    "rimraf": "^4.1.2",
     "typescript": "^4.9.4"
   },
-  "dependencies": {
-    "firebase": "^9.16.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+  "peerDependencies": {
+    "firebase": "^9",
+    "react": "^18",
```
Comment (sdodson99) [inline package/package.json:53]: playing it safe for now. everything probably works on v16 and v17? we'd have to test that eventually i guess.

File: package/package.json [hunk 2]
```diff
@@ -1,20 +1,36 @@
...
+    "dist/*"
+  ],
   "scripts": {
     "predev": "npm run clean",
     "dev": "parcel watch",
     "test": "jest",
     "build": "parcel build",
-    "clean": "rm -rf .parcel-cache"
+    "prepublish": "npm run clean && npm run build",
+    "publish": "npm publish --access=public",
```
Comment (sdodson99) [inline package/package.json:32]: we can probably automate this someday with a github action. that's a much lower priority right now though.

File: package/package.json (hunk 1 above)
Comment (edodso2) [reply package/package.json:53]: For the Firebase I am using the v9 API. Its much different then v8.

**Review by edodso2 — APPROVED** [review]
🚀

**Pull request merged**